budget/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "budget",
+  "version": "0.1.0",
+  "description": "AWS Budget",
+  "keywords": [
+    "AWS",
+    "Budget"
+  ],
+  "author": {
+    "name": "Meir Elbaz",
+    "email": "maornet@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/winglang/winglibs.git",
+    "directory": "budget"
+  },
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
yep, since they are needed for preflight execution

Reply:
Do I need to put everything in `peerDependencies`?

In `containers` lib for example I see:

```
"peerDependencies": {
    "cdktf": "*",
    "winglang": "*"
  },
  "dependencies": {
    "@cdktf/provider-aws": "^18.0.5",
    ...
  }
}
```

Should I also remove the version constraints?

Reply:
Yes, everything has to be peers, the containers library needs to be fixed as well.

If you take a dep on `winglang` or `cdktf`, use `*`. Otherwise specify a minimal version.